app/views.py

```diff
@@ -1268,34 +1273,27 @@ def folder_update(request, folder_id):
         return Response(settings, status.HTTP_200_OK)
```

Comment:
What is setting definition doing in a folder update endpoint?. 
Are you just copy pasting this blindly?